.eslintrc.js

```diff
@@ -125,6 +125,23 @@ module.exports = {
     'jsdoc/require-returns-type': 'error',
     'jsdoc/require-returns': 'error',
     'jsdoc/valid-types': 'error',
+
+    // TODO: To be reviewed:
```

Comment:
If I understand correctly, with the new version of the ESLint new rules are appeared? That rules cause ~2k problems which for now they are temporarily silent. Before merging I would look at each rule and solve it. Or we can create an issue and we can start working on it after merging this PR. WDYT?

Reply:
I'm all for the news issue. I disabled them temporarily, because I didn't want to make any decisions about them on my own, but they shouldn't all be disabled indefinitely for sure.

Reply:
I've created an issue for those rules and more https://github.com/handsontable/handsontable/issues/7531